docs/sources/writing-guide/topic-types/tutorial/index.md

```diff
@@ -96,6 +96,54 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
 
 If getting access to the tutorial data is complex, include the instructions in the steps of the tutorial. If getting access to the data is straightforward, include it in the "Before you begin" section.
 
+## Where to locate your tutorial
+
+The source for your tutorial should be in your project repo in a `tutorials` folder but it should be displayed on the main Grafana [Tutorials](/tutorials) page.  The source must be in your project repo to makes it easy for team members to review and edit the content. 
+
+Follow the steps below to hide the tutorial from your documentation's table of contents and display it on the Tutorials page.
+
+> Note: This procedure is for writers who have permissions to update the Grafana website repo.
+
+To publish your tutorial to the main Tutorials page:
+
+1. In your documentation repo, create a folder under `sources` called `tutorials` and add your tutorial to this folder.
+
+1. To hide your tutorial from your documentation table of contents, integrate the following to the `$.cascade` field in the YAML metadata of your `sources/_index.md` file. 
+
+    ```yaml
+    cascade:
+    - _target:
+        path: /docs/grafana-cloud/tutorials/**
+      _build:
+        list: false
+        render: false
+    - _target:
+        path: /docs/grafana-cloud/**
+    ```
+
+    > Create the `cascade` field if it does not exist. Substitute your repo for `grafana-cloud` in this example.
+
+1. To add your tutorial to the Tutorials page, integrate the following to the `$.manual_mounts` field in the `config/_default/config.yaml` file in the website repo:
```

Comment:
aren't we already in a task that describes how to add a tutorial to the tutorials page? I think maybe moving step two out into it's own section could help?